base/traits.jl

```diff
@@ -3,15 +3,26 @@
 ## numeric/object traits
 # trait for objects that have an ordering
 abstract type OrderStyle end
+
+"""
+    Ordered()
+
+Indicate that a type `T` guarantees that a method at most as specific as
+`isless(::T, ::T)` is defined and when called will not throw an exception.
```

Comment:
```suggestion
Indicate that a type `T` guarantees that a call `isa(::T, ::T)` will return `true` or `false`.
```

Reply:
Can you please edit the suggestion changing `isa` to `isless`? Thank you!

Reply:
I did it